app/models/project.rb

```diff
@@ -0,0 +1,66 @@
+class Project < ApplicationRecord
+
+    belongs_to :user
+    has_many :lead
+    has_many :leads, through: :lead, source: :user
```

Comment:
this code will be confuse, because the correct definition of has_many for leads is "has_many :leads", and if you want to get users linked to the project by leads you must use other name like "has_many :users_by_leads, through: :lead, source: :user"

Reply:
:white_check_mark: